.gitignore

```diff
@@ -25,6 +25,9 @@ wheels/
 *.egg
 MANIFEST
 
+# Pytorch
+*.pth
```

Comment:
Thanks! We can add custom rules at the bottom, which may be easier to check.

Reply:
Thanks for the fast review, made the changes and follow the same for the future PR. Cheers!

Reply:
.pth is saved to `work_dirs` by default, which has already been ignored.

Reply:
adding .pth gives user the flexibility to keep checkpoints any where in the project and also we dont need to explicitly specify or request them to keep checkpoints in specific directory. thanks!